react-effects/my-app/src/List.tsx

```diff
@@ -24,7 +24,7 @@ export function List() {
       }
     };
     fetchData();
-  }, [isLoading]);
+  }, [items]);
```

Comment:
Well, you don't use `items`, either! The `items` you have listed in your fetchData function is a local variable. It is not the items state variable. The `items` in your dependency array _is_ the items state variable. You don't use the items state variable in your fetchData function.

Reply:
OK, now you've made the code more clear, so now you can scan the fetchData function and see that you never use `items`. So it should not be a dependency. (It still is.)

Reply:
updated to only run once on mount.